<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pore_flow_dlg</class>
 <widget class="QWidget" name="pore_flow_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>孔板流量计算</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../usecad.qrc">
    <normaloff>:/images/cbm.ico</normaloff>:/images/cbm.ico</iconset>
  </property>
  <widget class="QWidget" name="widget_main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>341</width>
     <height>431</height>
    </rect>
   </property>
   <widget class="QLabel" name="mineName_label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;抽放的瓦斯混合量Q&lt;span style=&quot; vertical-align:sub;&quot;&gt;混&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="k">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>140</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="q_hun">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>300</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="x">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>100</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="dh">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>60</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="cacl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计 算</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保 存</string>
    </property>
   </widget>
   <widget class="QLabel" name="mineName_label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>孔板前后端所测压差Δh</string>
    </property>
   </widget>
   <widget class="QLabel" name="mineName_label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>实际孔板流量特性系数K</string>
    </property>
   </widget>
   <widget class="QLabel" name="mineName_label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>瓦斯管内测点温度t</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="b">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>180</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="delta_p">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>220</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="mineName_label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;抽放的瓦斯纯量Q&lt;span style=&quot; vertical-align:sub;&quot;&gt;纯&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="delta_t">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>260</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="mineName_label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>混合气体中瓦斯浓度X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="t">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>20</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="mineName_label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>气压校正系数δp</string>
    </property>
   </widget>
   <widget class="QLabel" name="mineName_label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>瓦斯浓度校正系数b</string>
    </property>
   </widget>
   <widget class="QLabel" name="mineName_label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>温度校正系数δT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="q_chun">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>340</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="k_cacl">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>140</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../usecad.qrc">
      <normaloff>:/images/udpate.png</normaloff>:/images/udpate.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="b_cacl">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>179</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../usecad.qrc">
      <normaloff>:/images/udpate.png</normaloff>:/images/udpate.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="delta_p_cacl">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>220</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../usecad.qrc">
      <normaloff>:/images/udpate.png</normaloff>:/images/udpate.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="delta_t_cacl">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>261</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../usecad.qrc">
      <normaloff>:/images/udpate.png</normaloff>:/images/udpate.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mineName_label_19">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>21</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(℃)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mineName_label_20">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>61</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(mmH&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;O)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="mineName_label_21">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>100</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(%)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="mineName_label_22">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>300</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(m&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;/min)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="mineName_label_23">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(m&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;/min)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_title" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QLabel" name="lab_Ico">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lab_Title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMenu_Close">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>t</tabstop>
  <tabstop>dh</tabstop>
  <tabstop>x</tabstop>
  <tabstop>k</tabstop>
  <tabstop>b</tabstop>
  <tabstop>delta_p</tabstop>
  <tabstop>delta_t</tabstop>
  <tabstop>q_hun</tabstop>
  <tabstop>q_chun</tabstop>
  <tabstop>cacl</tabstop>
  <tabstop>save</tabstop>
 </tabstops>
 <resources>
  <include location="../usecad.qrc"/>
 </resources>
 <connections/>
</ui>
